crates/router/src/core/mandate/utils.rs

```diff
@@ -0,0 +1,78 @@
+use std::marker::PhantomData;
+
+use common_utils::{errors::CustomResult, ext_traits::ValueExt};
+use diesel_models::Mandate;
+use error_stack::ResultExt;
+
+use crate::{
+    core::{errors, payments::helpers},
+    types::{self, api::mandates, domain, PaymentAddress},
+};
+const IRRELEVANT_PAYMENT_ID_IN_MANDATE_REVOKE_FLOW: &str =
+    "irrelevant_payment_id_in_mandate_revoke_flow";
+
+const IRRELEVANT_ATTEMPT_ID_IN_MANDATE_REVOKE_FLOW: &str =
+    "irrelevant_attempt_id_in_mandate_revoke_flow";
+
+const IRRELEVANT_CONNECTOR_REQUEST_REFERENCE_ID_IN_MANDATE_REVOKE_FLOW: &str =
+    "irrelevant_connector_request_reference_id_in_mandate_revoke_flow";
+
+pub async fn construct_mandate_revoke_router_data(
+    connector_name: &str,
+    request: mandates::MandateId,
```

Comment:
`connector_name` is already present in `merchant_account`, and `mandate_id` is present in `mandate`, these parameters can be removed from the function definition